### DISCUSSION TITLE
Any MAUI success stories? Please share your app link

### DISCUSSION THREAD

**Discussion by gautambjain** [dotnet/maui#17751] [General]
Have been reading only negative stuff about MAUI. Aren't there any success stories? Does anyone have a working usable app on App Store or Play Store? Please share the link and if possible share your MAUI experience.

**Comment by FM1973** [comment]
Yes we have a maui app in the playstore and appstore. It does work, but as I stated in a couple of other posts we are stuck at version 7.0.59 with a ton of workarounds for known bugs still not fixed. I will ask my customer if it is ok to post the link here. If they agree I will happily do so.

**Comment by Strypper** [comment]
https://github.com/Strypper/mauisland
on Windows Store also there's a link in the readme

**Comment by gautambjain** [comment]
Really... no one else has anything to share?

**Reply by Kirk-iOS** [reply]
Mine isn't released yet.

**Comment by giuseppenovielli** [comment]
https://github.com/dotnet/maui/discussions/15109#discussioncomment-5942697
https://github.com/dotnet/maui/discussions/9599
https://github.com/dotnet/maui/discussions/12019#discussioncomment-4385419

**Comment by ShariatPanah** [comment]
I created really a simple Countdown app with Maui and published it on CafeBazaar, here's the link to the app:
[https://cafebazaar.ir/app/com.lanternsoft.countdown?l=en](https://cafebazaar.ir/app/com.lanternsoft.countdown?l=en)

but I'm surprised by the final bundle size, it's 67MB for this simple app, just wow!!!!!!

**Comment by pvoelker** [comment]
I've been working on a simple game for Bible quizzing.  Got hooked on Lottie animations with this project.  I love the framework of Maui, coming from a WPF background.

However there have been so many problems.  I have known issues on .NET 7 Maui in Windows and iOS.  I can't go to Maui on .NET 8 because of new bugs.  I am **really** hoping things get sorted out with the final release on .NET 8 Maui.

[JBQ Quiz Me!](https://pvoelker.github.io/JBQQuizMe/)
